lib/_kernel.scm

```diff
@@ -4279,6 +4279,12 @@ end-of-code
             scheme-object   ;; device
    "___os_device_stream_open_process"))
 
+(define-prim ##os-device-open-raw
```

Comment:
I will remove the function if you feel strongly about it, and move it to the library.